cpp/daal/src/algorithms/dtrees/forest/classification/df_classification_predict_dense_default_batch_impl.i

```diff
@@ -213,6 +232,10 @@ protected:
     size_t _nClasses;
     size_t _cachedNClasses;
     VotingMethod _votingMethod;
+    size_t _blockSize;
+    size_t _minTreesForThreading;
+    size_t _minNumberOfRowsForVectSeqCompute;
+    float _scaleFactorForVectParallelCompute;
     static const size_t s_cMaxClassesBufSize = 32;
```

Comment:
This also looks like a hyperparameter. Can be target for a future work in this direction.

Reply:
Good catch.